script.js

```diff
@@ -117,6 +118,7 @@ const fetchStockInfo = async (ticker_symbol) => {
     const response = await fetch(requestUrl);
     const data = await response.json();
     stockObject = data;
+    // COMMENT(andymina);
```

Comment:
Since this function is async-await and you await the data on L119, you can actually just return `data`. This would be helpful for splitting the stockMath info function into two parts as mentioned above.

Reply:
Yeah, I can see why that would make sense since my reasoning for doing that was for an issue that I had before I used async. But can you elaborate as to how it would be useful for splitting the stockMath function into two? I thought that line of code is just redundant, I'm not seeing how it would be more helpful

Reply:
Returning would be more helpful and generally a bit cleaner because the logic for stockMath wouldn't have to be nested in a `.then()`. If it is nested its easier to track and debug in the future.